dubbo-common/src/test/java/org/apache/dubbo/common/utils/NetUtilsTest.java

```diff
@@ -180,4 +181,23 @@ public void testToURL() throws Exception {
         String url = NetUtils.toURL("dubbo", "host", 1234, "foo");
         assertThat(url, equalTo("dubbo://host:1234/foo"));
     }
+
+    @Test
+    public void testIsValidV6Address() {
+        System.setProperty("java.net.preferIPv6Addresses", "true");

```

Comment:
```
String saved = System.setProperty("java.net.preferIPv6Addresses");
System.setProperty("java.net.preferIPv6Addresses", "true");
...
System.setProperty("java.net.preferIPv6Addresses", saved);
```
This way is more elegant and will not break subsequent tests.

Reply:
String saved = System.getProperty("java.net.preferIPv6Addresses");
System.setProperty("java.net.preferIPv6Addresses", "true");
...
System.setProperty("java.net.preferIPv6Addresses", saved);

Reply:
Good catch, it is better.